blog/2022-08-30-q-and-a-august-2022-second-part.mdx

```diff
@@ -94,7 +94,7 @@ We don't have a way to block this bootloader completely right now, we have a lot
 
 We also have another challenge to come. Right now, Luos is an early technology. It is a state-of-the-art technology, so we have a lot of things moving around every day, and it is difficult for us to keep all the revisions compatible.
 
-It is still a little bit complex for us to do that, so to patch this, Viktoria is working on an application that can update the bootloader, allowing you to upgrade the Bluetooth directly into your board.
+It is still a little bit complex for us to do that, so to patch this, Viktoria is working on an application that can update the bootloader, allowing you to upgrade the Bluetooth directly onto your board.
```

Comment:
Same here ?

Reply:
We already have a nofollow link on this page about Viktoria